app/models/answer.rb

```diff
@@ -0,0 +1,12 @@
+class Answer < ActiveRecord::Base
+  belongs_to :question
+  belongs_to :user
+  has_many :comments, :as => :commentable
+  has_many :votes, :as => :votable
+
+  validates :content, :user_id, :question_id, presence: true
```

Comment:
In validations, prefer validating the object to the id. So ```validates :question``` would check that the id is present but also that it points at a valid question. ```validates :question_id``` only checks that question_id is not null.
``validates :content, :user, :question, presence: true```